Fixed build.gradle, gradle.properties and pom.xml

xzescha opened the pull request:
- Updated version of  "org.jetbrains.intellij" to version "1.17.4" in build.gradle
- Changed reference to right directory of eolang parser in build.gradle
- Updated qlice plugin addition in pom.xml
- Changed version of IntelliJ IDEA in gradle.properties

xzescha committed:
Fixed build.gradle and pom.xml

- Updated version of  "org.jetbrains.intellij" to version "1.17.4" in build.gradle
- Changed reference to right directory of eolang parser in build.gradle
- Updated qlice plugin addition in pom.xml
- Changed version of IntelliJ IDEA in gradle.properties

0crat commented:
@xzescha Our work policy states that non-compliant branch names incur a penalty. Your branch name \"`fixes_and_improvements`\" [doesn't follow our convention](https://www.yegor256.com/2014/04/15/github-guidelines.html), resulting in a **-12 point** deduction. Please remember to name branches after their corresponding ticket numbers. For this case, \"`60`\" would have been the correct name. Let's adhere to this practice to maintain consistency and clarity in our project management.

<!-- It is [not a good idea](https://www.yegor256.com/2014/04/15/github-guidelines.html) to name Git branches the way you named this one: "`fixes_and_improvements`". You've earned -12 points. Next time, better give your branch the same name as the number of the ticket that you are solving. In this case, a perfect name, for example, would be "`60`".  -->

xzescha committed:
Added improvements for cross-platform development

Some minor improvements

xzescha committed:
Update build.gradle

Changed filename

xzescha committed:
Minor improvements to fix Qulice errors

Commit:
Fixed handwritten files

xzescha committed:
Fixed handwritten files pt.2

xzescha committed:
Fixed tasks sequence and removed extra case in Highlighter

xzescha committed:
Hopefully, this will work fine

Added cleanParser task. I've tried to configure qulice to avoid auto-generated source code, but somehow plugin just can't do it.

xzescha committed:
Another update for build.gradle

Now an updated build config to run jacoco report

xzescha committed:
Update codecov.yml

Maybe now all tests will pass

xzescha committed:
Update codecov.yml pt.2

xzescha committed:
commit to try pipeline again

xzescha committed:
Merge branch 'fixes_and_improvements' of https://github.com/xzescha/eo-intellij-plugin into fixes_and_improvements

xzescha commented:
The one and only check that's failed for now is Codecov. I've created an issue for this. Please, check it out asap, because I assume that's not the problem that I can manage for now: https://github.com/objectionary/eo-intellij-plugin/issues/62

yegor256 commented on build.gradle:
```diff
@@ -23,34 +24,48 @@ java {
 apply plugin: 'jacoco'
 
+task cleanParser(type: Delete) {
+    delete fileTree("src/main/java/org/eolang/jetbrains/parser") // Удаляет папку с парсером
```
@xzescha don't write comments in Russian

xzescha replied:
Fixed. Hopefully, I've replaced them all

xzescha committed:
Removed Russian comments

I've removed comments in Russian. And replaced it with English translation

yegor256 commented:
@xzescha thanks!

0crat commented:
@xzescha Great contribution! 🌟 You've earned +4 points: +4 base, adjustments for code volume (-8), review process (-16), and comments (-4), with a +28 bonus. Your effort is appreciated! Keep [improving](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) and contributing – we're excited to see your next submission!

<!-- Thanks for the contribution! You've earned +4 points for this: +4 as a basis; -8 for too many hits-of-code (278 >= 200); -16 for the lack of code review; -4 for very few (4) comments; +28 to give you at least something. Please, [keep](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) them coming.  -->
